openrtb_ext/request.go

```diff
@@ -71,6 +71,15 @@ type ExtRequestPrebid struct {
 	// - basic: excludes debugmessages and analytic_tags from output
 	// any other value or an empty string disables trace output at all.
 	Trace string `json:"trace,omitempty"`
+
+	//
+	AdServerTargeting []AdServerTargeting `json:"adservertargeting,omitempty"`
```

Comment:
I would suggest a slight renaming, to avoid thing = list of things. Either `AdServerTargeting []AdServerTarget`, `AdServerTargeting []AdServerTargetRule`, or `AdserverTargets []AdServerTargeting`

Reply:
renamed to `AdServerTargeting []AdServerTarget`